pkg/session_recording/recordings.jsx

```diff
@@ -364,9 +414,11 @@
          */
         journalctlStart() {
             /* TODO Lookup UID of "tlog" user on module init */
-            var matches = ["_UID=987"];
-            var options = {follow: true, count: "all"};
-
+            /* Or Lookup by tlog-rec parameter */
+            // var matches = ["_COMM=tlog-rec"];
```

Comment:
This will go away, and we will be using tlog-rec-session only.
First, because Cockpit is a system-level, administration tool, and tlog-rec is an end-user tool, while tlog-rec-session is a system-level tool.
Second, because _COMM can be spoofed by the user, or lost by journald.